src/log_util.rs

```diff
@@ -30,9 +30,9 @@ extern "C" fn delegate(c_args: *mut gpr_log_func_args) {
         &Record::builder()
             .args(format_args!("{msg}"))
             .level(level)
-            .file(file_str.into())
+            .file_static(file_str.into())
```

Comment:
Is it really static?

Reply:
yes, file name is generated by macro, see https://github.com/tikv/grpc/blob/996605a5e62f3f00043ac8d3ebca84523bc2dd76/include/grpc/impl/codegen/log.h#L55